pkg/cassandra/config.go

```diff
@@ -56,7 +56,7 @@ func addNumTokens(template *DatacenterConfig, cfg *cassConfig) {
 	// explicitly set it because the config builder defaults to num_tokens: 1
 	if cfg.CassandraYaml.NumTokens == nil {
 		version := template.ServerVersion
-		if strings.ToLower(template.ServerType) == "cassandra" && version.Major() == 3 {
+		if template.ServerType == api.DistributionCassandra && version.Major() == 3 {
```

Comment:
A note on the naming.. here we have refer the Distribution with two different names. On the CRD it's "ServerType", yet on that enum it's "Distribution". This feels odd. Is it a type of server or is it just a different distribution of the same server?

Reply:
And does distribution mean deb-packaged Cassandra vs. tar-Cassandra? Perhaps an OSS Cassandra packaged by soem vendor like there used to be ?

Reply:
The fact that we're using "ServerType" in the CRD doesn't make Distribution irrelevant here. Also, the term distribution is not a matter of how a software is packaged, but rather how vanilla or not it is compared to the upstream project.
It makes sense in the Linux world, just as it seems to me it makes sense in the Cassandra world.

It's not a hill I'll die on though, so if this is a blocker for you I can change this to `ServerTypeDSE` and `ServerTypeCassandra`.

Reply:
@burmanm, what do you think?

Reply:
Also, if we use `ServerType` as type, we'll have the `ServerType` field that has the `ServerType` type 😕 not really a great situation. I think that's why I came up with an alternate name.